packages/terraform-modules-public/testnet/variables.tf

```diff
@@ -0,0 +1,271 @@
+}
+
+variable validator_account_passwords {
+  type        = list(string)
+  description = "Array with the Validator etherbase account passwords"
+}
+
+variable txnode_account_addresses {
+  type        = list(string)
+  description = "Array with the TX node etherbase account addresses"
+}
+
+variable txnode_private_keys {
+  type        = list(string)
+  description = "Array with the TX node etherbase account private keys"
+}
+
+variable txnode_account_passwords {
+  type        = list(string)
+  description = "Array with the TX node etherbase account passwords"
+}
+
+variable proxy_account_addresses {
+  type        = list(string)
+  description = "Array with the Proxy etherbase account addresses"
+}
+
+variable proxy_account_passwords {
+  type        = list(string)
+  description = "Array with the Proxy etherbase account passwords"
+}
+
+variable proxy_private_keys {
+  type        = list(string)
+  description = "Array with the Proxy etherbase account private keys"
+}
+
+variable proxy_enodes {
+  type        = list(string)
+  description = "Array list with the proxy enode address (without enode://)"
+}
+
+variable proxy_private_node_keys {
+  type        = list(string)
+  description = "Array with the Proxy node private keys"
+}
+
+variable bootnode_enode_address {
+  type        = string
+  description = "Network bootnode enode address (without enode://)"
+  default     = ""
+}
+
+variable bootnode_ip_address {
+  type        = string
+  description = "Network bootnode ip address"
+  default     = ""
+}
+
+variable deploy_txnode_lb {
+  type        = bool
+  description = "Deploy or not the Load Balancer for the TX Nodes"
+  default     = false
+}
+
+variable static_nodes_base64 {
+  type        = string
+  description = "Content of the genesis file encoded in base64"
+}
+
+variable reset_geth_data {
+  type        = bool
+  description = "Specifies if the existing chain data should be removed while creating the instance"
+}
+
+# Attestation service vars
+variable deploy_attestation_service {
+  type        = bool
+  description = "Deploy or not the Attestation Service"
+  default     = false
+}
+
+variable attestation_service_db_username {
+  type        = string
+  description = "The User for the database"
+  default     = "celo"
+}
+
+variable attestation_service_db_password {
+  type        = string
+  description = "The password for the database"
+  default     = "secret"
+}
+
+variable attestation_service_docker_image_repository {
+  type        = string
+  description = "The docker image repository for the attestation service"
+  default     = ""
+}
+
+variable attestation_service_docker_image_tag {
+  type        = string
+  description = "The docker image tag for the attestation service"
+  default     = ""
+}
+
+variable attestation_service_account_address {
+  type        = string
+  description = "The etherbase account address for signing the attestations"
```

Comment:
Misleading, this should be the account address of the validator

Reply:
Got it! thanks